src/components/index.js

```diff
@@ -1,5 +1,6 @@
 import CheckBoxField from './CheckBox';
 import TextInputField from './TextInput';
+import TextLink from './TextLink';
 import CustomButton from './CustomButton';
 
-export {CheckBoxField, TextInputField, CustomButton};
+export {CheckBoxField, TextInputField, CustomButton,TextLink};
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
export {CheckBoxField, TextInputField, CustomButton, TextLink};
```